recipe/meta.yaml

```diff
@@ -309,12 +316,6 @@ about:
 
       See the post [here](https://iscinumpy.gitlab.io/post/root-conda/) for more information about using this Conda package.
 
-      Caveats
-      =======
-
-      General
-      -------
-
       The ROOT package will prepare the required compilers (see below). Everything in Conda is symlinked into
```

Comment:
See below? This needs a little editing (since the compilers were listed under Linux and macOS sections and now are gone.